portal/src/main/webapp/js/src/oncoprint/webgl/src/js/oncoprinttrackoptionsview.js

```diff
@@ -192,6 +193,27 @@ var OncoprintTrackOptionsView = (function () {
 	    $dropdown.append($sort_dec_li);
 	    $dropdown.append($dont_sort_li);
 	}
+	if (model.isTrackExpandable(track_id)) {
+	    $dropdown.append($makeDropdownOption(
+		    (model.isTrackExpanded(track_id) ? 'More' : 'Show') + ' genes',
```

Comment:
Good point, missed that one. It has to be clear to the user what the biological meaning of the tracks is, but it's now pretty obvious that that information should be defined in the same place as the callback that encodes this meaning.